src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,25 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      className={`${completed ? 'TodoInfo TodoInfo--completed' : 'TodoInfo'}`}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+
+      {user && <UserInfo user={user} />}
```

Comment:
```suggestion
      {user && <UserInfo user={user} />}
```
Cool) but I got three rejects when I had to use that syntax)))) I hope there will not be here

Reply:
hope this conditional rendering in my case won't be rejected